src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
The Rectangle class should not extend Figure directly if Figure is intended to be an abstract class to define common behavior. Consider using an interface to represent shared behaviors or ensure that Figure is not an abstract class if inheritance is used correctly.